ztp/gitops-subscriptions/argocd/CoreReduction.md

```diff
@@ -0,0 +1,37 @@
+# Core Reduction with Workload Partitioning and Performance Profile
+
+Maximum number of cores can be allocated for customer workload only by pinning host and cluster workloads to a small number of cores. This is known as "core reduction" and be achieved by correctly configuring Workload Partitioning and Performance Profile.
+
+## Part 1 - Configure Performance Profile
```

Comment:
Maybe we should add a little intro blurb here that says the Performance Profile is the one that handles setting up services that run outside of the cluster for workload partitioning, things like `crio, systemd, kubelet`.

Reply:
Yeah I have it down below `This will pin all the host level services such as systemd and crio.` but can move it up